pkg/platform/kube/project/client.go

```diff
@@ -24,7 +24,7 @@ type Client struct {
 
 func NewClient(parentLogger logger.Logger, platformInstance platform.Platform, consumer *client.Consumer) (abstractproject.Client, error) {
 	newClient := Client{
-		Logger:   parentLogger.GetChild("projects-client"),
+		Logger:   parentLogger.GetChild("projects-client-kube"),
```

Comment:
```suggestion
		Logger:   parentLogger.GetChild("projects-kube"),
```
I think we can skip the word "client" in loggers (since mostly X is X-client)